packages/expo-font/src/FontTypes.web.ts

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Platform-specific extensions like `.web.ts` don't work with types. The declared types need to work on all platforms simultaneously.